requirements.txt

```diff
@@ -0,0 +1,9 @@
+geopandas
+geoplot
```

Comment:
This is extraneous:

1. The `geoplot` dependency was removed a long time ago. This is no longer even an optional depenedency.
2. This requirements file doesn't really serve a purpose. If you need a list of packages that this library requires, you can look at the `setup.py`. A set of package pins would be useful, but only if there is a CI environment that uses them (this library is simple enough that you don't really need it for development purposes).